packages/xo-server-sdn-controller/src/protocol/openflow-channel.js

```diff
@@ -0,0 +1,384 @@
+import createLogger from '@xen-orchestra/log'
+import ipaddr from 'ipaddr.js'
+import openflow from '@xen-orchestra/openflow'
+import util from 'util'
+import Stream from '@xen-orchestra/openflow/stream'
```

Comment:
Didn't we rename this?

Reply:
Yes i had the changes only locally ^^'
Pushed now.